_layouts/post.html

```diff
@@ -3,7 +3,7 @@
 ---
 
 <p class="meta">
-  {{ page.date | date: "%B %d, %Y" }} | Category: {{ page.categories | category_links }}
+  {{ page.date | date: "%B %d, %Y" }} | Category: {% for category in page.categories %} {{category}}{% endfor %}
```

Comment:
This was done as the previous version concatenated words together. 

The category_links doesn't create links, so I replaced it with an inline category listing.